src/acquisition/cdcp/cdc_extract.py

```diff
@@ -130,94 +130,99 @@ def store_result(cur, epiweek, state, num1, num2, num3, num4, num5, num6, num7,
-        num3 = get_num_hits(cur, epiweek, state, pages[2])
-        num4 = get_num_hits(cur, epiweek, state, pages[3])
-        num5 = get_num_hits(cur, epiweek, state, pages[4])
-        num6 = get_num_hits(cur, epiweek, state, pages[5])
-        num7 = get_num_hits(cur, epiweek, state, pages[6])
-        num8 = get_num_hits(cur, epiweek, state, pages[7])
-        total = get_total_hits(cur, epiweek, state)
-        store_result(cur, epiweek, state, num1, num2, num3, num4, num5, num6, num7, num8, total)
-        print(' %d-%s: %d %d %d %d %d %d %d %d (%d)' % (epiweek, state, num1, num2, num3, num4, num5, num6, num7, num8, total))
-      except Exception as ex:
-        print(' %d-%s: failed' % (epiweek, state), ex)
-        #raise ex
-      sys.stdout.flush()
-
-  # disconnect
-  cur.close()
-  if not test_mode:
-    cnx.commit()
-  cnx.close()
+    # page title templates
+    pages = [
+        "%What You Should Know for the % Influenza Season%",
+        "%What To Do If You Get Sick%",
+        "%Flu Symptoms & Severity%",
+        "%How Flu Spreads%",
+        "%What You Should Know About Flu Antiviral Drugs%",
+        "%Weekly US Map%",
+        "%Basics%",
+        "%Flu Activity & Surveillance%",
+    ]
+
+    # location information
+    states = sorted(cdc_upload.STATES.values())
+
+    # connect
+    u, p = secrets.db.epi
+    cnx = mysql.connector.connect(user=u, password=p, database="epidata")
+    cur = cnx.cursor()
+
+    # weeks to update
+    if first_week is None:
+        cur.execute("SELECT max(`epiweek`) FROM `cdc_extract`")
+        for (first_week,) in cur:
+            pass
+    if last_week is None:
+        cur.execute("SELECT max(`epiweek`) FROM `cdc_meta`")
+        for (last_week,) in cur:
+            pass
+    print("extracting %d--%d" % (first_week, last_week))
+
+    # update each epiweek
+    for epiweek in flu.range_epiweeks(first_week, last_week, inclusive=True):
+        # update each state
+        for state in states:
+            try:
+                num1 = get_num_hits(cur, epiweek, state, pages[0])
+                num2 = get_num_hits(cur, epiweek, state, pages[1])
+                num3 = get_num_hits(cur, epiweek, state, pages[2])
+                num4 = get_num_hits(cur, epiweek, state, pages[3])
+                num5 = get_num_hits(cur, epiweek, state, pages[4])
+                num6 = get_num_hits(cur, epiweek, state, pages[5])
+                num7 = get_num_hits(cur, epiweek, state, pages[6])
+                num8 = get_num_hits(cur, epiweek, state, pages[7])
+                total = get_total_hits(cur, epiweek, state)
+                store_result(
+                    cur, epiweek, state, num1, num2, num3, num4, num5, num6, num7, num8, total
+                )
+                print(
+                    " %d-%s: %d %d %d %d %d %d %d %d (%d)"
+                    % (epiweek, state, num1, num2, num3, num4, num5, num6, num7, num8, total)
+                )
```

Comment:
black is worse here but the actual fix is to refactor to something like

```suggestion
                nums = []
                for i in range(8):
                    nums[i] = get_num_hits(cur, epiweek, state, pages[i])
                total = get_total_hits(cur, epiweek, state)
                store_result(cur, epiweek, state, *nums, total)
                print(f" {epiweek}-{state}: {' '.join(str(n) for n in nums)} ({total})")
```